src/utils/wallets.ts

```diff
@@ -18,3 +21,31 @@ const isMMRejection = (err: Error & { code?: number }): boolean => {
 export const isWalletRejection = (err: Error & { code?: number }): boolean => {
   return isMMRejection(err) || isWCRejection(err) || isKeystoneError(err)
 }
+
+const WalletNames = {
+  METAMASK: 'MetaMask',
```

Comment:
You can import `ProviderLabel.MetaMask` from `'@web3-onboard/injected-wallets'` to get this.